CMakeLists.txt

```diff
@@ -9,5 +9,7 @@ else()
     message("Monero surce directory is not defined, using default ${MONERO_DIR}")
 endif()
 
+include_directories(/usr/local/include)
```

Comment:
This is clearly due to how you have built the project, it is incorrect to include system specifics like this.